portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,4 +32,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html;");
     response.getWriter().println("Hello Sam");
   }
+
+  public void updateDatastore(String nickname, String comment) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("nickname",nickname);
+    commentEntity.setProperty("comment",comment);
+    datastore.put(commentEntity);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    updateDatastore("placeholder_nickname","placeholder_comment");
```

Comment:
nit: add a space after ','

Reply:
Updated with commit 9617098

Reply:
there's still a space missing between `"placeholder_nickname",` and `"placeholder_comment"` on line 46